src/components/LeaderBoard/LeaderBoard.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import styles from './LeaderBoard.module.css';
+import history from '../../services/history';
+import { setPlayers } from './LeaderBoardActions';
+
+export class LeaderBoard extends React.Component {
```

Comment:
`Leaderboard` is one word I think?

Reply:
I googled it 👀

Reply:
Actually, you're right. but opinions on the matter are controversial